api/fossa/upload.go

```diff
@@ -116,3 +116,25 @@ func Upload(title string, locator Locator, options UploadOptions, data []SourceU
 
 	return ReadLocator(unmarshalled.Locator), nil
 }
+
+// UploadContributors posts contributor data to the project decribed by locator
+func UploadContributors(data map[string]string, locator Locator) {
+	payload, err := json.Marshal(data)
+	if err != nil {
+		log.Debugf("Failed to marshal contributor map: %s", err)
+	}
+	q := url.Values{}
+	q.Add("locator", locator.String())
+	endpoint, err := url.Parse("api/contributors?" + q.Encode())
+	if err != nil {
+		log.Debugf("Failed to generate contributor URL: %s", err)
+	}
+	body, statusCode, err := Post(endpoint.String(), payload)
+	if err != nil {
+		log.Debugf("Failed to POST contributor data: %s", err)
+	}
+	if statusCode != 200 {
+		log.Debugf("FOSSA reported an issue while uploading contributor data: %s", body)
+	}
+	return
+}
```

Comment:
Do we really want to continue this function on these errors? Seems strange to eg fail to encode data but then still try to POST it.

Reply:
Oops... forgot to add the early returns.